### DISCUSSION TITLE
Issue: DatePicker Component Rendering Difference Between PrimeNG 17 and 19 - Chrome

### DISCUSSION THREAD

**Discussion by renanhfranca** [primefaces/community#3249] [PrimeNG]
I’ve noticed that the DatePicker component in PrimeNG version 19 behaves differently compared to version 17. The rendering and interaction seem to have changed significantly.

This behavior can be observed directly on the demo page. You can check it out using the link below:
https://primeng.org/datepicker#basic

Version 17
![image](https://github.com/user-attachments/assets/ef6de71f-9889-47c7-9fb1-f34515abb9b0)

Version: 19
![image](https://github.com/user-attachments/assets/5ecd37b9-82e5-48b5-a144-7e7bc879d8e7)

Has anyone else encountered this issue or observed similar differences? Any insights would be appreciated!

**Comment by dbaines30** [comment]
I have seen this too. Tried a blank, standalone app and see the same issue there.

**Reply by dbaines30** [reply]
Looks like this has been fixed in @18.0.2
